simfection/simulation_day.py

```diff
@@ -55,7 +65,7 @@ def run(self):
         self.connection_engine.create_connections(cpp)
         self.anchor_tracker.end_anchor('connection')
         connection_runtime = self.anchor_tracker.timing('connection')
-        logging.debug(f'- Connections made in {connection_runtime} seconds')
+        logger.debug('- Connections made in {:.2f} seconds.'.format(connection_runtime))
```

Comment:
Nit - Since we're requiring Python3, we should try using the more readable F strings for formatting: https://realpython.com/python-f-strings/
Example:
logging.debug(f'- Connections made in {connection_runtime:.2f} seconds')